src/components/CanvasWrapper.tsx

```diff
@@ -236,10 +237,21 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
      * TODO: Margin calculation?
      */
     const setSize = (): void => {
+      // TODO: Potentially unsafe calculation here when using `window.innerWidth`. If close to the breakpoint where the side
+      // panel gets wrapped to below the canvas, the scrollbar added to account for the increased page height
+      // will cause this calculation to change (window.innerWidth will become smaller by ~15 pixels).
+      // Under certain circumstances, this can cause a flickering effect as the canvas resizes to accommodate the scrollbar,
+      // which causes the page to shrink, which causes the scrollbar to disappear, and so on in a loop.
+      // I've fixed this for now by setting the breakpoint to 1250 pixels, but it's not a robust solution.
+
```

Comment:
Example of what this looks like:

https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/50cdbb9b-1628-4800-8929-0bf17724c738